main.tf

```diff
@@ -295,6 +295,14 @@ resource "aws_cloudfront_distribution" "website_cdn_redirect" {
     bucket = aws_s3_bucket.website_logs.bucket_domain_name
     prefix = "${var.website-domain-redirect}/"
   }
+  dynamic "custom_error_response" {
```

Comment:
@hat-diag The Terraform module builds 2 CloudFront Distribution: `website_cdn_redirect` and `website_cdn_root`. You added this block only to the `website_cdn_redirect` distribution but I assume it's also required for the main one?

Reply:
@cloudmaniac sorry for the late response. Yes, it's actually my bad I put it in the wrong distribution, we won't need it for `website_cdn_redirect` as it needs to always return redirect response.
Let me update 🙏 .